dev/com.ibm.ws.microprofile.rest.client_fat_tck/fat/src/org/eclipse/microprofile/rest/client/tck/RestClientTckPackageTest.java

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.microprofile.rest.client.tck;
+
+import java.io.File;
+import java.nio.file.Files;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import componenttest.annotation.AllowedFFDC;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.MvnUtils;
+
+/**
+ * This is a test class that runs a whole Maven TCK as one test FAT test.
+ * There is a detailed output on specific
+ */
+@SuppressWarnings("restriction")
+@RunWith(FATRunner.class)
+public class RestClientTckPackageTest {
+
+    @Server("FATServer")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        server.startServer();
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        server.stopServer("CWMCG0007E", "CWMCG0014E", "CWMCG0015E", "CWMCG5003E", "CWWKZ0002E");
+    }
+
+    @Test
+    @AllowedFFDC // The tested deployment exceptions cause FFDC so we have to allow for this.
+    public void testRestClientTck() throws Exception {
+        if (!MvnUtils.init) {
+            MvnUtils.init(server);
+        }
+        // Everything under autoFVT/results is collected from the child build machine
+        File mvnOutput = new File(MvnUtils.home, "results/mvnOutput_TCK");
+        int rc = MvnUtils.runCmd(MvnUtils.mvnCliTckRoot, MvnUtils.tckRunnerDir, mvnOutput);
```

Comment:
Just for info https://github.com/OpenLiberty/open-liberty/pull/1725/commits/7472a8e3c0a0c9076047ac11a74dacd49996af2f is going to deliver a runTCKMvnCmd that
pushes some of the directory handling into the Utils.